tests/test_docopt.py

```diff
@@ -540,24 +543,41 @@ def test_long_options_error_handling():
     with raises(
         DocoptExit,
         match=r"Warning: found unmatched \(duplicate\?\) arguments.*--non-existent",
-    ):
+    ) as err:
         docopt("Usage: prog", "--non-existent")
+    exc = err.value
+    assert exc.collected == []
+    assert exc.left == [_Option(None, "--non-existent", 0, True)]
+
     with raises(
         DocoptExit, match=r"Warning: found unmatched \(duplicate\?\) arguments.*--ver\b"
-    ):
+    ) as err:
         docopt(
             "Usage: prog [--version --verbose]\n" "Options: --version\n --verbose",
             "--ver",
         )
+    exc = err.value
+    assert exc.collected == []
+    assert exc.left == [_Option(None, "--ver", 0, True)]
+
     # --long is missing ARG in usage
     with raises(DocoptLanguageError, match=r"unmatched '\('"):
         docopt("Usage: prog --long\nOptions: --long ARG")
-    with raises(DocoptExit, match=r"--long requires argument"):
+
+    with raises(DocoptExit, match=r"--long requires argument") as err:
         docopt("Usage: prog --long ARG\nOptions: --long ARG", "--long")
+    exc = err.value
+    assert exc.collected == []
```

Comment:
hmm, in an ideal world I wish`collected` actually contained something here, but it looks like there is no current way to represent this sort of half-baked option. I'm fine leaving for now.